Improve Windows Terminal caret tracking with delayed UIA updates (#20398)
nvaccess/nvda: user_docs/en/userGuide.md

On Windows versions before Windows 11 22H2, text in the console that is not visibly displayed in the window is not accessible with NVDA's text review commands.
Therefore, it is necessary to scroll the console window to read earlier text.
In newer versions of the console and in Windows Terminal, it is possible to review the entire text buffer freely without the need to scroll the window.

Windows Terminal may delay UIA caret updates, especially in remote sessions over SSH.
NVDA waits slightly longer for caret movement in Windows Terminal, so it is less likely to report stale characters.
If caret reporting is still inaccurate, try increasing [Caret move timeout](#AdvancedSettingsCaretMoveTimeout) in Advanced settings.

> Increasing this value may make caret movement feel slower, so consider changing it in a configuration profile for terminal applications.

Reviewer: Think you need to keep an empty line belone.

```suggestion
Increasing this value may make caret movement feel slower, so consider changing it in a configuration profile for terminal applications.

```

On Windows versions before Windows 11 22H2, text in the console that is not visibly displayed in the window is not accessible with NVDA's text review commands.
Therefore, it is necessary to scroll the console window to read earlier text.
In newer versions of the console and in Windows Terminal, it is possible to review the entire text buffer freely without the need to scroll the window.

> Windows Terminal may delay UIA caret updates, especially in remote sessions over SSH.

Reviewer 2: ```suggestion
In some cases, such as when connected over SSH, caret updates may be delayed.
```
